### PR TITLE
Add debug log for missing config

### PR DESCRIPTION
**Pull request by chrisduerr**
We currently log whenever we fall back to the default config because of
an error in the config itself. We also log when the config was
successfully loaded and where it was loaded from. The only scenario
where no config related message is logged is when there is no
configuration file present.

Logging this case should make it easier to debug issues like #3240,
without requiring any knowledge from maintainers about this edgecase.

### CODE REVIEW

File: alacritty/src/config/mod.rs [hunk 1]
```diff
@@ -121,6 +121,8 @@ pub fn installed_config() -> Option<PathBuf> {
                     return Some(fallback);
                 }
             }
+
+            debug!(target: LOG_TARGET_CONFIG, "No configuration file found");
```
Comment (kchibisov) [inline alacritty/src/config/mod.rs]: I feel like it should be on `info` level, since configuration loaded is also on `info` level.

**Comment by kchibisov** [comment]
Also, you should move logging there https://github.com/alacritty/alacritty/blob/767d59155a5d834d22ccb7c7d439b418ffad5404/alacritty/src/main.rs#L132 in `else` branch.

File: alacritty/src/config/mod.rs (hunk 1 above)
Comment (chrisduerr) [reply alacritty/src/config/mod.rs]: That's a great point, I just put it on the highest level that doesn't cause a UI error, but info makes more sense.

**Comment by chrisduerr** [comment]
> Also, you should move logging there 

That's a good point, that should make it so it works on Windows too.

**Pull request merged**